source/Patches/RainbowMod/PlayerTabPatch.cs

```diff
@@ -1,39 +1,63 @@
-﻿using System;
-using HarmonyLib;
+﻿using HarmonyLib;
+using Reactor.Extensions;
+using System;
 using UnityEngine;
 using UnityEngine.Events;
 using Object = UnityEngine.Object;
 
 namespace TownOfUs.RainbowMod
 {
-    [HarmonyPatch(typeof(PlayerTab), nameof(PlayerTab.OnEnable))]
+    [HarmonyPatch(typeof(PlayerTab))]
     public class PlayerTabPatch
     {
-        public static bool Prefix(PlayerTab __instance)
+        [HarmonyPatch(nameof(PlayerTab.OnEnable))]
+        [HarmonyPostfix]
```

Comment:
```suggestion
        [HarmonyPostfix]
        [HarmonyPatch(nameof(PlayerTab.OnEnable))]
```
just a style thing, looks better this way